docs/graphql/core/migrations/advanced/seed-data-migration.rst

```diff
@@ -35,7 +35,9 @@ Step 3: Write an insert statement
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 The next step is to write an insert statement that populates the database with
-seed data, like this:
+seed data.
+
+Let's consider the following schema:
```

Comment:
This is not a schema. The schema already exists, otherwise we couldn't make all these inserts. We can remove this.